Northstar.CustomServers/mod/scripts/vscripts/mp/_base_gametype_mp.gnut

```diff
@@ -420,11 +420,13 @@ void function DecideRespawnPlayer( entity player )
 
 void function RespawnAsPilot( entity player )
 {
+	if(IsAlive( player )) return
```

Comment:
Maybe add a comment to both functions so that it's clear for someone going over the script in the future that this is a hotfix?